### PR TITLE
implement PDP url for product variant

### PR DESCRIPTION
**Pull request by AlicjaSzu**
I want to merge this change because...

### Pull Request Checklist

1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

File: src/@next/components/organisms/ProductVariantPicker/types.ts [hunk 1]
```diff
@@ -10,4 +10,6 @@ export interface IProps {
   ) => void;
   selectSidebar?: boolean;
   selectSidebarTarget?: HTMLElement | null;
+  queryVariants?: { [key: string]: string };
```
Comment (dominik-zeglen) [inline src/@next/components/organisms/ProductVariantPicker/types.ts:13]: How about 

```suggestion
  queryVariants?: Record<string, string>;
```

Comment (dominik-zeglen) [reply src/@next/components/organisms/ProductVariantPicker/types.ts:13]: I'm not sure about making this optional too

File: src/views/Product/Page.tsx [hunk 2]
```diff
@@ -1,65 +1,100 @@
 import { smallScreen } from "../../globalStyles/scss/variables.scss";
 
 import classNames from "classnames";
-import * as React from "react";
+import { isEmpty } from "lodash";
+import { History } from "history";
```
Comment (dominik-zeglen) [inline src/views/Product/Page.tsx:5]: I'm not sure if this is a good layer to handle routing, personally I'd move it to the view

File: src/@next/components/organisms/ProductVariantPicker/types.ts [hunk 3]
```diff
@@ -10,4 +9,6 @@ export interface IProps {
   ) => void;
   selectSidebar?: boolean;
   selectSidebarTarget?: HTMLElement | null;
+  queryAttributes: Record<string, string>;
+  updateUrlWithAttributes: (slug: string, value: string) => void;
```
Comment (orzechdev) [inline src/@next/components/organisms/ProductVariantPicker/types.ts:13]: Name of this prop suggest that this component decides if url update should be performed. But from what I see it doesn't. If we change `updateUrlWithAttributes` to e.g. `onChangeAttribute`, the actual implementation on what to do when attribute with `slug` and `value` changes will be delegated to the higher order component, so component prop would be used in other scenarios, what now with `updateUrlWithAttributes` is logically possible, but semantically suggests the contrary. Am I right?

File: CHANGELOG.md [hunk 4]
```diff
@@ -34,6 +34,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Completely recreate checkout and cart, with new checkout nad cart SDK - #639 by @orzechdev, @mateuszkula
 - Hide payment options - #678 by @orzechdev
 - Fix unhandled JWT token expiration - #696 by @orzechdev
+- Implement PDP url for product variant - #708 by @AlicjaSzu
```
Comment (orzechdev) [inline CHANGELOG.md:37]: It's unclear for me what really has been changed.

Comment (orzechdev) [inline CHANGELOG.md:37]: It's unclear for me what really has been changed.

File: src/@next/components/organisms/ProductVariantPicker/types.ts (hunk 3 above)
Comment (AlicjaSzu) [reply src/@next/components/organisms/ProductVariantPicker/types.ts:13]: you're right. i'll find better name for this function

**Pull request merged**